schema/attribute/drivers-license.json

```diff
@@ -0,0 +1,15 @@
+{
+	"$id": "http://platform.selfkey.org/schema/attribute/drivers-license.json",
+	"$schema": "http://platform.selfkey.org/schema/identity-attribute.json",
+	"idAttribute": true,
+	"idAttributeRepository": "http://platform.selfkey.org/repository.json",
+	"title": "Drivers License",
+	"description": "A copy of an individuals drivers license.",
+	"type": "object",
+	"properties": {
+		"image": {
+			"$ref": "http://platform.selfkey.org/schema/document/image.json"
```

Comment:
A driver's license should have two images - one front, one back.

Reply:
will merge

Reply:
fixed